pkgs/applications/misc/waybar/default.nix

```diff
@@ -47,7 +47,7 @@ stdenv.mkDerivation rec {
   propagatedBuildInputs = lib.optionals withMediaPlayer [
     glib
     playerctl
-    python38Packages.pygobject3
+    python3Packages.pygobject3
```

Comment:
```suggestion
    python3.pkgs.pygobject3
```

Reply:
Nice, will add this change and remove python3Packages from the inputs entirely :)